Makefile

```diff
@@ -123,7 +123,7 @@ api:
 clean_restart_e2e_test: clean_e2e_tests build_release run_e2e_tests
 
 migrate_postgres:
-	migrate -database "postgres://arcuser:arcpass@localhost:5432/arcdb?sslmode=disable"  -path database/migrations/postgres  up
+	migrate -database "postgres://postgres:postgres@localhost:5432/postgres?sslmode=disable"  -path database/migrations/postgres  up
```

Comment:
The name of the DB is `blocktx`. So I think that connection string should be
`"postgres://postgres:postgres@localhost:5432/blocktx?sslmode=disable"`

Reply:
I'll extract those params to be configurable. Thanks for pointing out.